### PR TITLE
Add Ecosia icon

### LINKED ISSUE #11420: Request: Ecosia
### Brand Name

Ecosia

### Website

https://www.ecosia.org/

### Popularity Metric

Similiarweb Global Rank: [293](https://www.similarweb.com/website/ecosia.org)

### Official Resources for Icon and Color

- SVG: https://www.ecosia.org/static/icons/safari-pinned-tab.svg
- Color: `#008009`, from Wikimedia Commons & blog.ecosia.org

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by pontushenriksson**
image

**Issue:** closes #11420

**Popularity metric:**
Similiarweb Global Rank: [251](https://www.similarweb.com/website/ecosia.org)

### Checklist

* [x] I updated the JSON data in `_data/simple-icons.json`
* [x] I optimized the icon with SVGO or SVGOMG
* [x] The SVG ``viewbox`` is ``0 0 24 24``

### Description

- I picked that hex value since it was provided in the issue but I double-checked with the Wikimedia source.
- I used boxy-svg.com and the terminal svgo version to optimize it.
- On [this page](https://ecosia.helpscoutdocs.com/article/559-company-support-material) they say "Do not use our old logo, you can [download our new one from this page](https://www.dropbox.com/sh/qj75colsl86n5km/AADGnV7jm1s3BBQM3LBcVvbNa?dl=0)." This is for companies who post on social media but it could be worth checking into and they have other assets on this page as well.

### CODE REVIEW

File: slugs.md
Comment (adamrusted) [inline slugs.md:1]: Slugs doesn't need updating as part of your PR, it's automatically generated / updated when the releases occur.

Comment (pontushenriksson) [reply slugs.md:1]: Oh okay. I will remove that part of my commit and send a new PR without that update :)

**Review by pontushenriksson — COMMENTED** [review]
Reverted the slugs change so please review the PR again.

File: _data/simple-icons.json
Comment (uncenter) [inline _data/simple-icons.json:4626]:
```suggestion
            "hex": "5DD25E",
```

It's not the color in their favicon but `#5dd25e` looks like the primary green they are using (now?) on their website:

image

**Review by uncenter — CHANGES_REQUESTED** [review]
Icon looks good to me! Just this question about the color.

File: _data/simple-icons.json
Comment (uncenter) [reply _data/simple-icons.json:4626]: Where are you finding that dropbox press kit?

Comment (uncenter) [reply _data/simple-icons.json:4626]: It's not in their press kit or on their website?

Comment (uncenter) [reply _data/simple-icons.json:4626]: > the text logo is not their “icon” 

Sure but we don't just use "icons", we use their logo. If we have a press kit available and it doesn't include this logo, we should use the logo they do provide. It seems like they rebranded from the tree icon two years ago but I guess kept some of it around?

Comment (uncenter) [reply _data/simple-icons.json:4626]: Do you have a vector version of it besides an outdated favicon file?

Comment (pontushenriksson) [reply _data/simple-icons.json:4626]: I mean that as we can't find an official use of the favicon in addition to the [safari-pinned-tab.svg](https://www.ecosia.org/static/icons/safari-pinned-tab.svg) I suggest that we use the wordmark logo displayed here https://commons.wikimedia.org/wiki/File:Logo_Ecosia_2022.svg that is used in the navbar on the main page and in their brand guidelines as a png.

image

**Comment by pontushenriksson** [comment]
Just let me know what version you would like to add. This would be the new preview:

image

**Comment by uncenter** [comment]
Please don't update the branch unnecessarily, everyone involved gets a notification when you do - maintainers reviewing will update it before merging to make sure the tests still pass and all that.

**Comment by pontushenriksson** [comment]
Oh okay. Sorry. It just said that it failed and I needed to update to the latest version in the actions popup, and I wanted to share the new preview image to make it easier for everyone.

File: _data/simple-icons.json
Comment (uncenter) [reply _data/simple-icons.json]: Sorry to raise this again but I think you were originally right! Let's bring back `#008009` here as that's the color used in the Wikimedia Commons file (unless we have any objections/other reasons?).

File: icons/ecosia.svg
Comment (uncenter) [inline icons/ecosia.svg:1]: I'm seeing some large differences between this version and the Wikimedia Commons file:

CleanShot 2024-07-24 at 16 39 07@2x

**Review by uncenter — CHANGES_REQUESTED** [review]

File: icons/ecosia.svg
Comment (pontushenriksson) [reply icons/ecosia.svg:1]: That is because I used the logo they are using on their own website ecosia.org but I could change it to the version on Wikimedia Commons if you rather want that.

Comment (uncenter) [reply icons/ecosia.svg:1]: That's fine but then you need to update the `source` property in the new entry to match that.

Comment (pontushenriksson) [reply icons/ecosia.svg:1]: Oh okay, sorry I forgot to change that when I changed the logo. I referenced the Wikimedia Commons file because I thought they were the same and as the SVG I used is not a file on the website but rather just added in the HTML and therefore I was unsure what I would link to, but I guess I reference ecosia.org there then.

Comment (uncenter) [reply icons/ecosia.svg:1]: Yep just link to the homepage (as per https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines).

**Comment by pontushenriksson** [comment]
Any updates on this icon? Is everything correct or should I change anything else?

**Comment by uncenter** [comment]
> Any updates on this icon? Is everything correct or should I change anything else?

Possibly, haven't gotten around to reviewing yet.

**Comment by uncenter** [comment]
Did you see my comment https://github.com/simple-icons/simple-icons/pull/11461#discussion_r1690395833?

**Comment by pontushenriksson** [comment]
Okay. Yes I read that but that was the color used in the Wikimedia Commons logo but as I used the logo that they are using on their website instead I picked the hex color they are using there. I could change it to the one from Wikimedia but then it won't match with the source I took the logo from.

File: _data/simple-icons.json
Comment (PratyushJayachandran) [inline _data/simple-icons.json:4627]:
```suggestion
            "source": "https://www.ecosia.org",
            "guidelines": "https://www.ecosia.org/imprint"
```
Their website seems to have been updated

**Review by PratyushJayachandran — APPROVED** [review]
Thank you for your contribution @pontushenriksson 🎉

**Pull request merged**
